code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -2110,3 +2080,48 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			return fixed_mut_color
 
 	return null
+
+/// Add species appropriate body markings
+/datum/species/proc/add_body_markings(mob/living/carbon/human/hooman)
+	for(var/datum/bodypart_overlay/simple/body_marking/markings as anything in body_markings) //loop through possible species markings
+		markings = new markings() // made to die... mostly because we cant use initial on lists but its convenient and organized
```

Comment:
please just put this in separate variable, this is not a hot path for it to matter